pkg/controller/eventprocessor/processor.go

```diff
@@ -38,9 +37,39 @@ func ProcessEvents(conf *config.Config) {
 			// Update user created groups.
 			controller.UpdateCustomGroups(data, groups, conf.Groupcrdclient)
 
+			// Persist in dgraph
+			PersistPayloads(data)
```

Comment:
Is it better to first persist data in dgraph then notify subscribers and update custom groups?

Reply:
Changed the order.